query_optimizer/logical/CreateIndex.hpp

```diff
@@ -59,18 +61,32 @@ class CreateIndex : public Logical {
    */
   const std::string& index_name() const { return index_name_; }
 
+  /**
+   * @return The list of attributes to build index upon.
+   */
+  const std::vector<expressions::AttributeReferencePtr>& index_attributes() const {
+    return index_attributes_;
+  }
+
+  /**
+   * @return Shared pointer to the index properties.
+   */
+  const std::shared_ptr<const IndexSubBlockDescription> index_description() const {
```

Comment:
Remove the first `const`, and I believe you borrowed the implementation for `CreateTable`, but I think we don't need the first one.